File: dart/2348.dart
class Solution {
    int zeroFilledSubarray(List<int> nums) {
        int count = 0, sac = 0;
        for (int i in nums)
            if (i == 0)
                sac += ++count;
            else
                count = 0;
        return sac;
    }
}

File: python3/2348.py
class Solution:
    def zeroFilledSubarray(self, nums: List[int]) -> int:
        sac = 0
        count = 0
        for i in nums:
            if i == 0:
                count += 1
                sac += count
            else:
                count = 0
        return sac
